docs/static/CNAME

```diff
@@ -1 +0,0 @@
-bootstrap-vue.js.org
```

Comment:
This is still needed for the bootstrap-vue.js.org domain validation (it is the only thing that needs this file)